src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "What's new in Astro - December 2023"
+description: ""
+publishDate: "December 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-november-2023/og.webp"
+lang: "en"
+---
+
+## Releases
+
+Just in time for the end of the year, we released Astro 4.0!
+
+- [Astro 4.0](https://github.com/withastro/astro/releases/tag/astro%404.0.0)
+- [Starlight 0.15](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.15.0)
+
+## Docs
+
+@TODO:
+
+## Community
+
+### Community Awards November
+
+- 🌟 $250 awarded to 
+- 🥳 $250 awarded to 
+- 🎉 $250 awarded to 
+- 🍾 $250 awarded to 
+- 🎊 $250 awarded to 
+- 🎁 $250 awarded to 
+
+## Showcase
+
+### Content
```

Comment:
Last month's section for this, we turned these more into small, descriptive sentences and used emoji. Let's follow that pattern here, too!